modules/docs/arrow-docs/docs/docs/effects/async/README.md

```diff
@@ -87,9 +87,23 @@ Behind the scenes `continueOn()` starts a new coroutine and passes the rest of t
 
 The function `continueOn()` is also available inside [`Monad Comprehensions`]({{ '/docs/patterns/monad_comprehensions' | relative_url }}).
 
+#### effect
+
+Similar to `MonadDefer`'s `later`, this constructor takes a single suspended function and optionally the `CoroutineContext` it has to be run on.
+
+```kotlin
+IO.async().run {
+  // In current thread
+  effect(CommonPool) {
```

Comment:
I would much rather not have this constructor and only have to teach `continueOn` which is more explicit and requires no nesting.